pkgs/by-name/ma/mackup/package.nix

```diff
@@ -3,29 +3,19 @@
   python3Packages,
   fetchFromGitHub,
   procps,
-  fetchpatch2,
 }:
 python3Packages.buildPythonApplication rec {
   pname = "mackup";
-  version = "0.8.40";
+  version = "0.8.41";
   pyproject = true;
 
   src = fetchFromGitHub {
     owner = "lra";
     repo = "mackup";
-    rev = "refs/tags/${version}";
-    hash = "sha256-hAIl9nGFRaROlt764IZg4ejw+b1dpnYpiYq4CB9dJqQ=";
+    rev = "${version}";
```

Comment:
This should stay as refs/tags/... is guaranteed to be a tag. Just using plain version could also be a branch.